lib/platform/gitea/gitea-helper.ts

```diff
@@ -379,7 +379,7 @@ export async function searchIssues(
   params: IssueSearchParams,
   options?: GiteaHttpOptions
 ): Promise<Issue[]> {
-  const query = queryParams(params).toString();
+  const query = queryParams({ ...params, type: 'issue' }).toString();
```

Comment:
Why do we need this? Can you link the API docu please.

Reply:
https://try.gitea.io/api/swagger#/issue/issueListIssues

```
	// - name: type
	//   in: query
	//   description: filter by type (issues / pulls) if set
	//   type: string
```

```go
	var isPull util.OptionalBool
	switch ctx.Query("type") {
	case "pulls":
		isPull = util.OptionalBoolTrue
	case "issues":
		isPull = util.OptionalBoolFalse
	default:
		isPull = util.OptionalBoolNone
	}
```
`OptionalBoolNone` here means that both are included

Reply:
Yes, it's weird that `/issues` API can fetch pulls (and does so by default) but backwards-compatibility...